src/clarity-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -147,6 +147,10 @@
         .datagrid-head {
             background-color: $clr-thead-bgcolor;
             border-bottom: 2*$clr-table-borderwidth solid $clr-light-midtone-gray;
+
+            .datagrid-row-actions {
+                padding: 10px 12px 11px;
```

Comment:
Do you think we should create variables for these values with meaningful names so that other developers who work with this stylesheet change these values while maintaining the vertical rhythm?

Reply:
I think that would be a larger refactor to ensure that changes to the vertical height remain consistent. Making this a variable doesn't have a lot of value IMO, because its a particular quirk to get the buttons exactly sized and positioned.

Reply:
I think the key to creating meaningful variables in SASS is identifying what about them makes them "quirks". In this example...

Reply:
Leaving comment so github will let me request changes.

Reply:
Replicated what I found elsewhere in cells.

```
padding: $clr-table-topcellpadding $clr-table-cellpadding $clr-table-bottomcellpadding;
```